api/src/main/java/com/codeforcommunity/dto/site/AddSiteRequest.java

```diff
@@ -0,0 +1,182 @@
+
+  public AddSiteRequest(
+      Boolean treePresent,
+      String status,
+      String genus,
+      String species,
+      String commonName,
+      String confidence,
+      Double diameter,
+      Double circumference,
+      Boolean multistem,
+      String coverage,
+      String pruning,
+      String condition,
+      Boolean discoloring,
+      Boolean leaning,
+      Boolean constrictingGate,
+      Boolean wounds,
+      Boolean pooling,
+      Boolean stakesWithWires,
+      Boolean stakesWithoutWires,
+      Boolean light,
+      Boolean bicycle,
+      Boolean bagEmpty,
+      Boolean bagFilled,
+      Boolean tape,
+      Boolean suckerGrowth,
+      String siteType,
+      String sidewalkWidth,
+      Double siteWidth,
+      Double siteLength,
+      String material,
+      Boolean raisedBed,
+      Boolean fence,
+      Boolean trash,
+      Boolean wires,
+      Boolean grate,
+      Boolean stump,
+      String treeNotes,
+      String siteNotes,
+      Integer blockId,
+      BigDecimal lat,
+      BigDecimal lng,
+      String city,
+      String zip,
+      String address) {
+    super(
+        treePresent,
+        status,
+        genus,
+        species,
+        commonName,
+        confidence,
+        diameter,
+        circumference,
+        multistem,
+        coverage,
+        pruning,
+        condition,
+        discoloring,
+        leaning,
+        constrictingGate,
+        wounds,
+        pooling,
+        stakesWithWires,
+        stakesWithoutWires,
+        light,
+        bicycle,
+        bagEmpty,
+        bagFilled,
+        tape,
+        suckerGrowth,
+        siteType,
+        sidewalkWidth,
+        siteWidth,
+        siteLength,
+        material,
+        raisedBed,
+        fence,
+        trash,
+        wires,
+        grate,
+        stump,
+        treeNotes,
+        siteNotes);
+    this.blockId = blockId;
+    this.lat = lat;
+    this.lng = lng;
+    this.city = city;
+    this.zip = zip;
+    this.address = address;
+  }
+
+  public Integer getBlockId() {
+    return blockId;
+  }
+
+  public void setBlockId(Integer blockId) {
+    this.blockId = blockId;
+  }
+
+  public BigDecimal getLat() {
+    return lat;
+  }
+
+  public void setLat(BigDecimal lat) {
+    this.lat = lat;
+  }
+
+  public BigDecimal getLng() {
+    return lng;
+  }
+
+  public void setLng(BigDecimal lng) {
+    this.lng = lng;
+  }
+
+  public String getCity() {
+    return city;
+  }
+
+  public void setCity(String city) {
+    this.city = city;
+  }
+
+  public String getZip() {
+    return zip;
+  }
+
+  public void setZip(String zip) {
+    this.zip = zip;
+  }
+
+  public String getAddress() {
+    return address;
+  }
+
+  public void setAddress(String address) {
+    this.address = address;
+  }
+
+  @Override
+  public List<String> validateFields(String fieldPrefix) throws HandledException {
+    String fieldName = fieldPrefix + "add_site_request.";
+    List<String> fields = new ArrayList<>();
```

Comment:
You might want to set this to `super.validateFields()`

Reply:
Done, set it to `super.validateFields("")`